platform/MCU/nRF52840/inc/platform_config.h

```diff
@@ -35,9 +35,7 @@
 #if PLATFORM_ID == PLATFORM_ARGON || \
```

Comment:
See: https://github.com/particle-iot/device-os/pull/2011/files#r373967791

Reply:
Same. I'm going to leave this as-is because it's a header in nRF52840 platform specific folder.